### PR TITLE
Sectors performance

### PR DESCRIPTION
**Pull request by gil27**
Hello, guys. This PR address issue #31 
I've tried to follow the code style, and be simple.

By the way, you guys have a good code base. 👍 👍 👍 

The following code implements support to fetch sectors performance from
api.

It implements the feature inside `lib/iex/api/sectors`.

I'm looking forward your review

### CODE REVIEW

**Comment by dblock** [comment]
Looks good. See bot comments above about the TOC and CHANGELOG.

File: README.md [hunk 1]
```diff
@@ -304,6 +305,19 @@ earnings.symbol_id # 4563
 See [#earnings](https://iextrading.com/developer/docs/#earnings) for detailed documentation or [earnings.rb](lib/iex/resources/earnings.rb) for returned fields.
 
 
+### Get Sector Performance
+
+Fetches earnings for a symbol.
+
+```ruby
+earnings = IEX::Resources::Earnings.get('MSFT')
```
Comment (rodolfobandeira) [inline README.md]: @gil27 This should be: `sectors = IEX::Resources::Sectors.get('MSFT')`

File: lib/iex/resources/sectors.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+require_relative 'base'
+
+module IEX
+  module Resources
+    class Sectors < Resource
+      property 'name'
```
Comment (rodolfobandeira) [inline lib/iex/resources/sectors.rb]: I would add `type` here as well. 

```
{
"type": "sector",
"name": "Materials",
"performance": 0.0047,
"lastUpdated": 1538491953067
}
```

File: README.md (hunk 1 above)
Comment (gil27) [reply README.md]: Shame on me 😭

**Comment by gil27** [comment]
Guys, I have been beat by `danger` here, could you guys help me out here to figure out what is the problem, I am missing something that I dont know what it is, for sure it is something dummy tough :/

**Comment by rodolfobandeira** [comment]
Hey @gil27 

Please try this:

`curl -o https://gist.githubusercontent.com/rodolfobandeira/cffea602a9727b6a9817f1d0195591a1/raw/5805ba093fd58b5a9db8e8febff8bcb718498cdf/fix-danger.patch`

`patch -p1 < fix-danger.patch`

`git commit -am 'Fix danger'; git push'`

File: README.md [hunk 3]
```diff
@@ -304,6 +306,20 @@ earnings.symbol_id # 4563
 See [#earnings](https://iextrading.com/developer/docs/#earnings) for detailed documentation or [earnings.rb](lib/iex/resources/earnings.rb) for returned fields.
 
 
+### Get Sector Performance
+
+Fetches earnings for a symbol.
```
Comment (rodolfobandeira) [inline README.md]: Missed this one. It should be something like: `Fetches latest sector's performance.`

**Comment by dblock** [comment]
Your missing a colon after the issue number in CHANGELOG:

```
* [#32](https://github.com/dblock/iex-ruby-client/pull/32): Add `IEX::Resource::Sectors` - [@gil27](https://github.com/gil27).
```

**Comment by dblock** [comment]
@rodolfobandeira you can merge at will, no need to ask me ;)

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,7 +1,10 @@
-### 0.4.4 (Next)
-
+### 0.5.0 (next)
 * Your contribution here.
 
+### 0.4.4 (2018/10/03)
```
Comment (dblock) [inline CHANGELOG.md]: We haven't released 0.4.4, please undo this change.

File: README.md [hunk 5]
```diff
@@ -304,6 +306,20 @@ earnings.symbol_id # 4563
 See [#earnings](https://iextrading.com/developer/docs/#earnings) for detailed documentation or [earnings.rb](lib/iex/resources/earnings.rb) for returned fields.
 
 
+### Get Sector Performance
+
+Fetches latest sector's performance.
+
+```ruby
+sectors = EX::Resources::Sectors.get('MARKET')
```
Comment (rodolfobandeira) [inline README.md]: `IEX` instead of `EX`

**Comment by rodolfobandeira** [comment]
@gil27 Just one last small thing I found going through the last check. 

` sectors = EX::Resources::Sectors.get('MARKET')` (Change it to `IEX`).

Once this goes in, we're good to merge!
Thanks again @gil27 !

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,9 +1,10 @@
-### 0.4.4 (Next)
-
+### 0.4.4 (next)
 * Your contribution here.
+* [#32](https://github.com/dblock/iex-ruby-client/pull/32): Add `IEX::Resource::Sectors` - [@gil27](https://github.com/gil27).
 
 ### 0.4.3 (2018/08/18)
 
+* Your contribution here.
```
Comment (dblock) [inline CHANGELOG.md]: That one *has* shipped, I'll edit in master.
